doc/users/next_whats_new/2020-05-26-cl-subplot.rst

```diff
@@ -0,0 +1,17 @@
+Constrained layout now supports subplot and subplot2grid
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Previously ``constrained_layout`` depended on a single ``GridSpec``
+for each logical layout on a figure, whereas ``plt.subplot`` added
+a new ``GridSpec`` for each call.  Now ``plt.subplot`` attempts to
```

Comment:
Constrained layout *still* depends on a single Gridspec, doesn't it?  Maybe change to "Previously, plt.subplot added a new GridSpec with each call, but constrained layout depends on working with a single GridSpec for...figure. Now, plt.subplot..."